### PR TITLE
FOR - 2019 - Add speaker

### PR DESCRIPTION
**Pull request by gb8may**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

### CODE REVIEW

File: content/events/2019-/speakers.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
++++
+Title = "Speakers"
```
Comment (bridgetkromhout) [inline content/events/2019-/speakers.md]: This directory path is not valid - it should have the city name too.

**Comment by gb8may** [comment]
I did by script... Is anything wrong with the add_speaker script?


*Mayara Gouveia*

DevOps Engineer | Information Security Specialist

*https://idevops.cloud <https://idevops.cloud>*

Linux User #566363 | LPI ID: LPI000348552 | Puppet ID: PPT0005371



Em sex, 20 de set de 2019 às 16:43, Bridget Kromhout <
notifications@github.com> escreveu:

**Comment by bridgetkromhout** [comment]
So, I see a number of problems - probably the main one is this:

This branch is 6 commits ahead, 357 commits behind devopsdays:master.

I strongly recommend rebasing. I can try to fix this PR, but most of your PRs will probably have trouble if you're this far behind.

**Comment by bridgetkromhout** [comment]
Okay, I've fixed this PR so I'll merge it, but please follow these directions before the next time you send a PR: https://github.com/devopsdays/devopsdays-web/blob/master/CONTRIBUTING.md#editing-your-site

Specifically, please do this:
```
git checkout master
git pull upstream master --rebase
git push
```

**Pull request merged**